tests/util/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+BACKEND=${1:-"linode:"}
+ITERATIONS=${2:-10}
+OUTPUT_FILE=${3:-"${PWD}/benchmark.out"}
+SUCCESSFUL_EXECUTIONS=0
+
+rm -rf "$OUTPUT_FILE"
+
+for i in $(seq 1 "$ITERATIONS"); do
```

Comment:
From a quick check I think that `seq` is already 1-based and you don't need the extra argument